### ISSUE TITLE
"File not found" error message

### ISSUE THREAD

**Issue by ms609** [mbrazeau/morphy.archive#7]
I made a subtle typo in the file name in `open=`.  The error message read 

`Successfully closed file...` 

Which suggested that the file had been opened, then closed – not that it didn't exist. 
Is it possible to include a "File not found" error, for clarity?

**Comment by mbrazeau** [comment]
Can do. Did any other file read errors result downstream from this?

**Comment by ms609** [comment]
Nope, I was able to open the (correct) file normally and run analysis with anticipated results.
